docs/Manual/05 Targets.md

```diff
@@ -18,11 +18,11 @@ We can also define custom targets:
 my_target = acc.Target(name="Custom processor", category=acc.Target.Category.CPU, architecture=acc.Target.Architecture.X86_64, family="Broadwell", extensions=["MMX", "SSE", "SSE2", "SSE3", "SSSE3", "SSE4", "SSE4.1", "SSE4.2", "AVX", "AVX2", "FMA3"], num_cores=22, num_threads=44, frequency_GHz=3.2, turbo_frequency_GHz=3.8, cache_sizes=[32, 256, 56320], cache_lines=[64, 64, 64])
 ```
 
-One benefit of targets is that they provide a standard way of accessing useful constants. For example, we may want to split an iteration space dimension by the number of elements that fit in a vector register.
+One of the benefits of the targets is that they provide a standard way of accessing useful constants. For example, we may want to split an iteration space dimension by the number of elements that fit in a vector register.
```

Comment:
"One benefit of targets" or "One of the benefits of targets" (the former is pretty clear to me)

Reply:
The second one indicates that there are multiple benefits. Although the first one also gives the same sense but with a relatively low emphasis. I recommend using the latter if targets have different applications.

Reply:
Yep, the first one is more accurate because the primary application of targets is to provide a standard way of accessing constants for the hardware that the user is trying to compile their package for.